compute_tools/src/pg_helpers.rs

```diff
@@ -63,12 +63,7 @@ impl GenericOption {
     /// Represent `GenericOption` as configuration option.
     pub fn to_pg_setting(&self) -> String {
         if let Some(val) = &self.value {
-            let name = match self.name.as_str() {
-                "safekeepers" => "neon.safekeepers",
-                "wal_acceptor_reconnect" => "neon.safekeeper_reconnect_timeout",
-                "wal_acceptor_connection_timeout" => "neon.safekeeper_connection_timeout",
-                it => it,
-            };
```

Comment:
This is unrelated to this PR, right?
Do we know for sure that all tenants have these updated?

Reply:
I've checked that in the console code we don't set them for quite a while, but this is a good point that we probably should check console db too. Will put a TODO, double check and remove later in a separate PR